packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -44,6 +44,8 @@ import { store as coreStore } from '@wordpress/core-data';
  */
 import { store as editPostStore } from '../../store';
 
+const isGutenbergPlugin = process.env.IS_GUTENBERG_PLUGIN ? true : false;
```

Comment:
@ntsekouras @Mamaduka Is this the correct way to do it?

Reply:
I saw the linting rule about ternary.. That seems good, yes.